blockchain/indexers/slpindex_test.go

```diff
@@ -0,0 +1,138 @@
+package indexers
+
+import (
+	"bytes"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/gcash/bchd/chaincfg/chainhash"
+	"github.com/gcash/bchd/wire"
+	"github.com/simpleledgerinc/goslp"
+	"github.com/simpleledgerinc/goslp/v1parser"
+)
+
+// TestSlpMesesageUnitTests downloads SLP parser unit tests and checks the parser throws for each test where code != nil
+func TestSlpMessageUnitTests(t *testing.T) {
```

Comment:
Is this test necessary? Seems to be testing SLP parsing but if it's being handled by the higher level library it seems this test should be in that library not BCHD right?

Reply:
Ok, deleted since this test is in goslp package.